apps/condo/bin/generate-tickets.js

```diff
@@ -171,7 +177,12 @@ class TicketGenerator {
 }
 
 const createTickets = async () => {
-    const TicketManager = new TicketGenerator({ ticketsByDay: { min: 20, max: 50 } })
+    const [organizationId] = process.argv.slice(2)
```

Comment:
I think it's better to pass `organizationId` to `createTicket`

Reply:
I've followed already existing approach to use class and assignments to `this`.
I would implement everything as functions, because here there is no use cases for classes as instances of things, with some life cycle.